src/app/shared/stack-blitz/stack-blitz-writer.ts

```diff
@@ -90,41 +94,66 @@ export class StackBlitzWriter {
-      description,
-      template: PROJECT_TEMPLATE,
-      tags: PROJECT_TAGS,
-    }, {openFile});
+  private _openStackBlitz({
+    title,
+    description,
+    openFile,
+    files,
+  }: {
+    title: string;
+    description: string;
+    openFile: string;
+    files: FileDictionary;
+  }): void {
+    stackblitz.openProject(
+      {
+        title,
+        files,
+        description,
+        template: PROJECT_TEMPLATE,
+        tags: PROJECT_TAGS,
+      },
+      {openFile}
+    );
   }
 
   /**
    * Builds an in-memory file dictionary representing an CLI project serving
    * the example. The dictionary can then be passed to StackBlitz as project files.
    */
   private async _buildInMemoryFileDictionary(
-      data: ExampleData, exampleId: string, isTest: boolean): Promise<FileDictionary> {
+    data: ExampleData,
+    exampleId: string,
+    isTest: boolean
+  ): Promise<FileDictionary> {
     const result: FileDictionary = {};
     const tasks: Promise<unknown>[] = [];
     const liveExample = EXAMPLE_COMPONENTS[exampleId];
-    const exampleBaseContentPath =
-      `${DOCS_CONTENT_PATH}/${liveExample.module.importSpecifier}/${exampleId}/`;
+    const exampleBaseContentPath = `${DOCS_CONTENT_PATH}/${liveExample.module.importSpecifier}/${exampleId}/`;
 
     for (const relativeFilePath of TEMPLATE_FILES) {
-      tasks.push(this._loadFile(TEMPLATE_PATH + relativeFilePath)
-        // Replace example placeholders in the template files.
-        .then(content => this._replaceExamplePlaceholders(data, relativeFilePath, content, isTest))
-        .then(content => result[relativeFilePath] = content));
+      tasks.push(
+        this._loadFile(TEMPLATE_PATH + relativeFilePath)
+          // Replace example placeholders in the template files.
+          .then(content =>
+            this._replaceExamplePlaceholders(data, relativeFilePath, content, isTest)
+          )
+          .then(content => (result[relativeFilePath] = content))
+      );
     }
 
     for (const relativeFilePath of data.exampleFiles) {
-      tasks.push(this._loadFile(exampleBaseContentPath + relativeFilePath)
-        // Insert a copyright footer for all example files inserted into the project.
-        .then(content => this._appendCopyright(relativeFilePath, content))
-        .then(content => result[`src/app/${relativeFilePath}`] = content));
+      // Note: Since we join with paths from the example data, we normalize
+      // the final target path. This is necessary because StackBlitz does
+      // not and paths like `./bla.ts` would result in a directory called `.`.
+      const targetPath = normalizePath(`src/app/${relativeFilePath}`);
+
+      tasks.push(
+        this._loadFile(exampleBaseContentPath + relativeFilePath)
+          // Insert a copyright footer for all example files inserted into the project.
+          .then(content => this._appendCopyright(relativeFilePath, content))
+          .then(content => (result[targetPath] = content))
```

Comment:
Only change in this file is the new `targetPath` variable in 149 and it being used here for `result[targetPath']`